sdk/identity/identity/review/identity.api.md

```diff
@@ -288,10 +296,15 @@ export interface ManagedIdentityCredentialResourceIdOptions extends TokenCredent
     resourceId: string;
 }
 
+// @public
+export interface MultiTenantTokenCredentialOptions extends TokenCredentialOptions {
+    additionallyAllowedTenants?: string[];
```

Comment:
`additionallyAllowedTenants` vs `additionalAllowedTenants`? We already have `additionalAllowedHeaderNames` and `additionalAllowedQueryParameters`

Reply:
This was a name that was decided by the rest of the identity team including @schaabs

Reply:
That's unfortunate that we will have inconsistency, but no strong opinion from me.